src/language_server.ts

```diff
@@ -209,11 +299,14 @@ export class LanguageServer {
 	 * @returns {Promise<any>} - A Promise that resolves to the result of `installLanguageServer()` after the server is installed.
 	 * If the user declines to install the server, returns a Promise that rejects with an error message.
 	 */
-	async proposeInstallLS(diagnoseId: string) {
+	async proposeInstallLS(diagnoseId: string, reason: LanguageServerDiagnoseResult) {
 		if (!this.pythonPath || !fileOrDirectoryExists(this.pythonPath)) {
 			await this.selectInterpreter(diagnoseId);
 		}
-		const response = await window.showErrorMessage(`Inmanta Language Server not installed, run "${this.pythonPath} -m pip install inmantals" ?`, 'Yes', 'No');
+		const msg = reason === LanguageServerDiagnoseResult.wrongLanguageServer
+		? "Wrong version of the Inmanta Language Server installed. Do you want to update? "
```

Comment:
```suggestion
		? "A new version of the Inmanta Language Server is available. Do you want to update? "
```
Mostly a suggestion I don't feel too strongly about. The only actual change request here is to not use "wrong version".

Side note: would be cool if we could get some users to use this flow for #1067, not sure how feasible that would be.

Reply:
It is easy :p just install the wrong version of inmantals in your venv by hand and start the extension. I've tried it using the .vsix file.